### PR TITLE
Update shields

### LINKED ISSUE #265: Add link to RTFD
At the moment it is hard to realize you can find documentation for edalize at https://edalize.readthedocs.io/en/latest/

### PR DESCRIPTION
**Pull request by umarcor**
Close #265.

### CODE REVIEW

**Comment by imphil** [comment]
Thanks! I like this update of the README.

Two comments:

* I don't think this update closes #265: Can you either remove the "Closes" comment, or add a textual description to the README about where to find the documentation?
* I also don't think we need the badges in the RTD documentation itself, but I'll leave the final word on that to @olofk.

**Comment by umarcor** [comment]
> I don't think this update closes [Add link to RTFD #265](https://github.com/olofk/edalize/issues/265): Can you either remove the "Closes" comment, or add a textual description to the README about where to find the documentation?

The updated shield pointing to RTFD includes the URL, so that's the first thing that users will see:

image

Moreover, @olofk or maybe yourself can add the URL at the top of the sidebar:

image

image

Nevertheless, I now added a textual reference at the top of the readme.

> I also don't think we need the badges in the RTD documentation itself, but I'll leave the final word on that to @olofk.

That was my interpretation of @mithro's "*maybe use the GitHub repository URL*". I understood that using the GitHub repository URL makes sense in the docs mostly.

File: README.rst [hunk 1]
```diff
@@ -21,6 +21,8 @@
 Edalize
 =======
 
+Find the docs at `edalize.rtfd.io <https://edalize.rtfd.io>`__.
```
Comment (imphil) [inline README.rst:24]: Can you put this comment in the beginning of the section "How to use it?". And docs -> documentation.

File: README.rst [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
    :target: https://gitter.im/librecores/edalize?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
-.. image:: https://github.com/olofk/edalize/workflows/CI/badge.svg
-        :target: https://github.com/olofk/edalize/actions?query=workflow%3ACI
-        :alt: CI status
-
-.. image:: https://readthedocs.org/projects/edalize/badge/?version=latest
-        :target: https://edalize.readthedocs.io/en/latest/?badge=latest
-        :alt: Documentation Status
+.. image:: https://www.librecores.org/olofk/edalize/badge.svg?longCache=true&style=flat-square
```
[the remark this answers is not in the document]
Comment (umarcor) [reply README.rst]: Done.

File: README.rst (hunk 1 above)
Comment (umarcor) [reply README.rst]: Done, by the way, I added a textual ref to this repo in section "Install it".

**Review by imphil — COMMENTED** [review]
LGTM, thanks!

**Pull request merged**